python-backend/app/api/variances/resources/variance_resource.py

```diff
@@ -43,24 +60,107 @@ def get(self):
 
-        filtered_query = apply_filters(
-            Variance.query.order_by(desc(Variance.received_date)),
-            [{
+        conditions = []
+        if args["application_status"] is not None:
+            status_filter_values = args["application_status"].split(',')
+            conditions.append({
+                'model': 'Variance',
                 'field': 'variance_application_status_code',
                 'op': 'in',
                 'value': status_filter_values
-            }])
+            })
+        if args["compliance_codes"] is not None:
+            compliance_codes_values = args["compliance_codes"].split(',')
+            conditions.append({
+                'model': 'Variance',
+                'field': 'compliance_article_id',
+                'op': 'in',
+                'value': compliance_codes_values
+             })
+
+        if args["expiry_date_before"] is not None:
+            conditions.append({
+              'model': 'Variance',
+              'field':  'expiry_date',
+              'op': '<=',
+              'value': args["expiry_date_before"]
+            })
+        if args["expiry_date_after"] is not None:
+            conditions.append({
+              'model': 'Variance',
+              'field': 'expiry_date',
+              'op': '>=',
+              'value': args["expiry_date_after"]
+            })
+        if args["issue_date_before"] is not None:
+            conditions.append({
+              'model': 'Variance',
+              'field': 'issue_date',
+              'op': '<=',
+              'value': args["issue_date_before"]
+            })
+        if args["issue_date_after"] is not None:
+            conditions.append({
+              'model': 'Variance',
+              'field': 'issue_date',
+              'op': '>=',
+              'value': args["issue_date_after"]
+            })
+
+        if args["major"] is not None:
+            conditions.append({
+                'model': 'Mine',
+                'field': 'major_mine_ind',
+                'op': '==',
+                'value': args["major"]
+            })
+        if args["search_terms"] is not None:
+            search_term_list = args["search_terms"].split(' ')
+            search_conditions = []
+            for search_term in search_term_list:
+                search_conditions.append(
+                    {
+                        'model': 'Mine',
+                        'field': 'mine_name',
+                        'op': 'ilike',
+                        'value': '%{}%'.format(search_term)
+                    })
+                search_conditions.append(
+                    {
+                        'model': 'Mine',
+                        'field': 'mine_no',
+                        'op': 'ilike',
+                        'value': '%{}%'.format(search_term)
+                    },
+                )
+            conditions.append(
+                {
+                    'or': search_conditions
+                })
```

Comment:
Consider checking out how I handled the "or" condition for search terms in the NOW Applications. There's a `or_` function that works really well.

https://github.com/bcgov/mds/pull/936/files#diff-c3f49332dad6bb23875dae032ba7ea71R83

Reply:
So I picked the convention I did because it matched the one that had already been implemented in this resource.  I can't see a way of implementing or_ without rewriting the entire pattern on this resource.